src/client/client.repository.ts

```
@@ -132,4 +132,52 @@ export class ClientRepository {
+        // Return array of authorities
+        return client.ExternalAuthority.map((auth) => auth.authority);
+      })
+      .catch((error) => {
+        if (error instanceof PrismaClientKnownRequestError) {
+          this.logger.error(`unknown database error`);
+          throw new InternalServerErrorException('unknown database error');
+        }
+        this.logger.error(`unknown error`);
+        throw new InternalServerErrorException('unknown error');
+      });
+    }
```

Biome

[error] Duplicate class member name "getAuthoritiesByClientUuid"



(lint/suspicious/noDuplicateClassMembers)